tests/tantivy_test.py

```diff
@@ -531,3 +532,27 @@ def test_query_from_json_field(self):
         # )
         # result = index.searcher().search(query, 2)
         # assert len(result.hits) == 1
+
+
+@pytest.mark.parametrize('bytes_kwarg', [True, False])
+@pytest.mark.parametrize('bytes_payload', [
+    b"abc",
+    bytearray(b"abc"),
+    memoryview(b"abc"),
+    BytesIO(b"abc").read(),
+    BytesIO(b"abc").getbuffer(),
+])
+def test_bytes(bytes_kwarg, bytes_payload):
+    schema = SchemaBuilder().add_bytes_field("embedding",).build()
```

Comment:
Nitpick, does this `,` here need to be there?

```suggestion
    schema = SchemaBuilder().add_bytes_field("embedding").build()
```

Reply:
oops, that made it over from the issue example